proposals/project-smoot.md

```diff
@@ -0,0 +1,136 @@
+## Smoot: An Open Source Project for DLT Interoperability 
+
+A proposal for LFDT 
+
+# Proposor(s)
+
+- Weijia Zhang (Wanchain) [weijia@wanchain.com](mailto:weijia@wanchain.com)  
+- Other organizations (TBD)
+
+# Abstract
+
+This project proposal outlines the plan to build an open source bridge for chain-to-chain interactions, based on the Enterprise Ethereum Alliance (EEA)’s DLT Interoperability Specification, as an open source project for the Linux Foundation’s Decentralized Trust(LFDT).
+
+# Context
+
+The EEA's DLT Interoperability Specification is a standard developed by members of the Enterprise Ethereum Alliance. Several implementations have been created to connect different blockchains based on this specification, including those by Wanchain, Adhara, QualitaX, and Fnality. We aim to develop an open-source project for implementing bridges that build interoperable components connecting both EVM and non-EVM blockchains.
+
+# Dependent Projects
+
+The Smoot project team intends to leverage technologies developed by LFDT, such as Cacti, Harmonia, and Besu EVM projects:
+
+* Besu   
+  Besu has been a popular EVM client node for both public and private blockchains. Besu was initially an implementation of the Enterprise Ethereum Alliance (EEA) Client Architecture and runs on both public and private networks. Project Smoot has a scope of work to connect to networks built with Besu clients.  
+    
+* Harmonia, a Hyperledger Lab  
+  Harmonia implements the Enterprise Ethereum Alliance's DLT Interoperability Specification's transaction messaging methods. Project Smoot will focus on implementing the event messaging method. The Smoot team will also explore the possibility of combining Project Smoot with Harmonia.  
+    
+* Hyperledger Cacti  
+  Cacti is an interoperability project that builds routing mechanisms for cross-chain minting and burning of assets. There are some common components that can be developed with various interoperability projects. For example, the cross-chain network identifier can be standardized so that different projects can use the same specification for blockchain identities.
```

Comment:
Do you have a document to discuss the gaps within the interoperability space? I am interested in visualizing how these different projects complement each other.

Reply:
Our EEA InterOp specification lists multiple mechanisms for blockchain interop and contains a table showing the characteristic (and limitations) of each mechanism for visualization.  The link of the EEA specification is in the reference section of the proposal.